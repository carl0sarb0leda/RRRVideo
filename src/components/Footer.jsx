import React from 'react';
import '../assets/styles/components/Footer.scss';

const Footer = () => (
	<footer className="footer">
		<a href="/">Terms and Conditions</a>
		<a href="/">Privacy Policy</a>
		<a href="/">Help Center</a>
	</footer>
);

export default Footer;
