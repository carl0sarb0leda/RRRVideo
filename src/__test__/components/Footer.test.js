import React from 'react'
import { mount } from 'enzyme'
import Footer from '../../client/components/Footer'

describe('Testing <Footer/>', () => {
    const footer = mount(<Footer />)
    test('Render Footer Component', () => {
        expect(footer.length).toEqual(1)
    })
    test('Render titles', () => {
        let globalFooter = 'Terms and Conditions Privacy Policy Help Center'
        expect(footer.find(".footer").text().replace(/\s/g, '')).toEqual(globalFooter.replace(/\s/g, ''))
    })
})