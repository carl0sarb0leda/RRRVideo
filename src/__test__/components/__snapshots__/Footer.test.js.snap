// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Testing UI Footer 1`] = `
<footer
  className="footer"
>
  <a
    href="/"
  >
    Terms and Conditions
  </a>
  <a
    href="/"
  >
    Privacy Policy
  </a>
  <a
    href="/"
  >
    Help Center
  </a>
</footer>
`;
